subprojects/ide/src/main/java/org/gradle/plugins/ide/internal/resolver/NullGradleApiSourcesResolver.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.plugins.ide.internal.resolver;
+
+import java.io.File;
+
+public class NullGradleApiSourcesResolver implements GradleApiSourcesResolver {
```

Comment:
nit: We could probably have a static instance for this resolver to avoid creating new instances. I doubt it has any impact so it's really up to you how you feel about this.